src/languages/php.js

```diff
@@ -11,38 +11,38 @@ Category: common
  * @returns {LanguageDetail}
  * */
 export default function(hljs) {
-  var VARIABLE = {
+  let VARIABLE = {
```

Comment:
Nice, but most (all?) of these should instead be const. :-)

Reply:
Okay.